examples/spring-reactive/README.md

```diff
@@ -0,0 +1,25 @@
+# Spring Reactive + CloudEvents sample
+
```

Comment:
Can you add a little description explaining the `RestController` with the two endpoints and `CloudEventHandlerConfiguration`?